src/general/world_effects/CompoundDiffusionEffect.cs

```diff
@@ -27,14 +26,12 @@ public void OnTimePassed(double elapsed, double totalTimePassed)
         HandlePatchCompoundDiffusion();
     }
 
-    private static (float Ambient, float Density) CalculateWantedMoveAmounts(Patch sourcePatch, Patch adjacent,
+    private static (float Ambient, float Density) CalculateWantedMoveAmounts(Patch adjacent,
         KeyValuePair<Compound, BiomeCompoundProperties> compound)
     {
         // Apply patch distance to diminish how much to move (to make ocean bottoms receive less surface
         // resources like oxygen)
-        // TODO: improve the formula here as sqrt isn't the best
-        float moveModifier = Constants.COMPOUND_DIFFUSE_BASE_MOVE_AMOUNT /
-            MathF.Sqrt(Constants.COMPOUND_DIFFUSE_BASE_DISTANCE + Math.Abs(sourcePatch.Depth[0] - adjacent.Depth[0]));
+        float moveModifier = Constants.COMPOUND_DIFFUSE_BASE_MOVE_AMOUNT;
```

Comment:
Just using a flat modifier here makes it so that there's nothing preventing oxygen from going to the deepest ocean very fast now, which is one of the major reasons I added the distance modifier. Maybe instead we could rewrite the code to only use the distance modifier if the distance is like at least 500?

Reply:
The number of patches between the epipelagic and sea floor already do that pretty well, but based on my quick research even that isn't accurate. As Buckly mentioned in the last Thrivestream, oxygen is actually pretty well spread in the modern oceans: https://en.wikipedia.org/wiki/Oxygen_minimum_zone

Reply:
Based on what I'm seeing, Oxygen in the bathypelagic or abyssopelagic should be around a third of the surface, while in this current code it's more like a tenth at best

Reply:
I guess I have to accept that as much as I think that will lead to players submitting bug reports I have to then read (as I'm almost the only one dealing with bug reports)... Though the comment that was left above is then obviously incorrect as it no longer related to the changed code.